packages/flutter_hooks/test/use_focus_node_test.dart

```diff
@@ -128,14 +139,17 @@ void main() {
     await tester.pumpWidget(
       HookBuilder(builder: (_) {
         focusNode = useFocusNode(
-          debugLabel: 'Bar',
-          onKey: onKey2,
-        );
+            debugLabel: 'Bar', onKey: onKey2, onKeyEvent: onKeyEvent2);
         return Container();
       }),
     );
 
     expect(focusNode.onKey, onKey, reason: 'onKey has no setter');
+    expect(
+      focusNode.onKeyEvent,
+      onKeyEvent,
```

Comment:
That likely should be onKeyEvent2. The event handlers are mutable now, so hooks should update them

Reply:
I updated both onKey and onKeyEvent to support the ability to mutate them.